### ISSUE TITLE
Add NPCs from Action Squad

### ISSUE THREAD

**Issue by Taberone** [Taberone/DoorKickers_NPCVarietyPack#1]
Try to add some NPCs from DK: Action Squad

Ex. Melee enemies (blue blade / red blade)
Dogs? (Requires someone to do sprites)
Machinegun dude
Bosses? (Ex. flamethrower guy)
Molotov throwers

### **Machete:**
Charges directly at troopers with a machete. Low damage, hits fast. Runs fast. No armor.
Wears blue.

**Partially done. To do: Recolor to blue, add speed**

### **Mad Machete:**
Same as Blue, but swings slower, hits harder. Runs slower than Blue, but faster than normal enemies. 
Wears red.

### **Molotov Manny:**
Throws molotovs. 
Need to look at Grenadier Glenn's code to see how to handle AI's throwing grenades.
Need to mod in molotov cocktails.

### **Leader Lemmy (Hostage taker guy):**
Junkie Harry with a proper pistol, possibly .45 caliber. Likes to hide behind hostages (if possible to implement).

### **Fanatic Frankie (Hostage Executioner):**
Junkie Harry with a proper pistol, possibly 9mm or .40 caliber. Likes to execute hostages if he spots a trooper.

### **Blowup Bart:**
Runs at your Troopers with fullspeed, then activates the bombs on his vest. Suicide bomber.

### **Flamer Finn:**
Has a flamethrower.

### **Shield Boss:**
Carries a shield and a spiked club.

### **Jacked Up Jones:**
???
I need to see him in action by playing more Action Squad first. He's in the files, but I have no idea what he does or how he behaves.

### **Wounded Officer:**
Basically a friendly Pointman.
